packages/test-sync/src/commonTest/kotlin/io/realm/kotlin/test/mongodb/common/CredentialsTests.kt

```diff
@@ -369,7 +369,7 @@ class CredentialsTests {
             payload = mapOf("mail" to TestHelper.randomEmail(), "id" to 0)
         )
 
-        assertFailsWithMessage<AuthException>("unauthorized") {
+        assertFailsWithMessage<AuthException>("Authentication failed") {
```

Comment:
Hmm. Weird that this has to go the other way. Was updated on the server two weeks ago and seemed pretty consistent with `unauthorized` throughout the server 🤔 But I guess it is ok, since it is just a test assertion 🤷